mock/py/mock.py

```diff
@@ -619,9 +647,231 @@ def unshare_namespace(config_opts):
             else:
                 sys.exit(e.resultcode)
 
+@traceLog()
+def chain():
+    mockconfig_path = '/etc/mock'
+
+    def createrepo(path):
+        if os.path.exists(path + '/repodata/repomd.xml'):
+            comm = ['/usr/bin/createrepo_c', '--update', path]
+        else:
+            comm = ['/usr/bin/createrepo_c', path]
```

Comment:
This would be nice to merge with config_opts["createrepo_command"] and post_build()